cli/config/configfile/file.go

```diff
@@ -43,6 +43,7 @@ type ConfigFile struct {
 	NodesFormat          string                      `json:"nodesFormat,omitempty"`
 	PruneFilters         []string                    `json:"pruneFilters,omitempty"`
 	Proxies              map[string]ProxyConfig      `json:"proxies,omitempty"`
+	BuildStreamContext   *bool                       `json:"buildStreamContext,omitempty"`
```

Comment:
Argh. naming. hm

Do we have other contexts other than for build? For booleans, I always liked, e.g. `enableSomething` (`EnableContextStreaming` e.g.), but that may be a bit too verbose

Do we expect more build-related options? If so, possibly have a `build` key, containing `stream-context` (in the JSON)